src/IterableCodeExtractor.php

```diff
@@ -111,7 +111,7 @@ protected static function calculateMatchScore( SplFileInfo $file, array $matcher
 		$root_relative_path = str_replace( static::$dir, '', $file->getPathname() );
 
 		foreach ( $matchers as $path_or_file ) {
-			$pattern = preg_quote( str_replace( '*', '__wildcard__', $path_or_file ) );
+			$pattern = preg_quote( str_replace( '*', '__wildcard__', $path_or_file ), '/' );
```

Comment:
This disregards the bug reported in #117 - MBString regexes don't use delimiters and using `preg_quote()` on a regex to be passed to one of the MBString regex functions is risky and potentially buggy.

Reply:
@jrfnl Noted. We have the bug report open, so the issue won't be forgotten. I assume it is a bigger refactor to fix this, though. We're not worse off for now with the above code, as the bug was detected and recorded for a later fix.